libs/core/testing/src/performance.cpp

```diff
@@ -15,10 +15,46 @@
 #include <type_traits>
 #include <vector>
 
+#if defined(HPX_HAVE_NANOBENCH)
+#define ANKERL_NANOBENCH_IMPLEMENT
+#include <nanobench.h>
+#endif
+
 namespace hpx::util {
 
     namespace detail {
 
+#if defined(HPX_HAVE_NANOBENCH)
+        constexpr int nanobench_epochs = 24;
+        constexpr int nanobench_warmup = 40;
+
+        char const* nanobench_hpx_template() noexcept
+        {
+            return R"DELIM({
+    "outputs (with nanobench)": [
+{{#result}}        {
+            "name": "{{name}}",
+            "executor": "{{context(executor)}}",
+            "series": [
+                {{#measurement}}{{elapsed}}{{^-last}},
+                {{/-last}}{{/measurement}}
+            ]
+        }{{^-last}},{{/-last}}
+{{/result}}    ]
+})DELIM";
```

Comment:
Yes. I think then we could make use of an command line option in the benchmarks to invoke the internal template.

Reply:
It may be possible to do this with a global variable through the hpx command line arguments facilities, e.g. without having to modify benchmarks to pass the template as an explicit variable, I will get back to you when I find out.

Reply:
Ohh, so something like `hpx:threads`

https://github.com/STEllAR-GROUP/hpx/blob/master/libs/full/command_line_handling/src/command_line_handling.cpp#L248-L289

Reply:
> It may be possible to do this with a global variable through the hpx command line arguments facilities, e.g. without having to modify benchmarks to pass the template as an explicit variable, I will get back to you when I find out.

I have added a `hpx:verbose_bench` command line option in https://github.com/STEllAR-GROUP/hpx/pull/6448/commits/dd126c65ef92216dbe6c36b2598ca6ae5c68f268 for the same. Here is the corresponding CI run testing the same: [test-run](https://github.com/STEllAR-GROUP/hpx/actions/runs/8310076266/job/22742156342?pr=6448)